projects/oled_demo/Makefile

```diff
@@ -0,0 +1,11 @@
+# Uncomment to use the desired platform
+# PLATFORM = xilinx
+# PLATFORM = altera
+# PLATFORM = aducm3029
+# PLATFORM = stm32
```

Comment:
If this doesn't work for anything besides xilinx, maybe we should only have that option.